shoop/core/api/orders.py

```diff
@@ -6,26 +6,37 @@
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
 from django.utils.timezone import now
-from rest_framework.serializers import ModelSerializer
+from rest_framework import serializers, status
+from rest_framework.decorators import detail_route
+from rest_framework.response import Response
 from rest_framework.viewsets import ModelViewSet
 
 from shoop.core.models import MutableAddress, Order, OrderLine, OrderStatus
+from shoop.utils.money import Money
 
 
-class OrderLineSerializer(ModelSerializer):
+class OrderLineSerializer(serializers.ModelSerializer):
     class Meta:
         model = OrderLine
 
 
-class AddressSerializer(ModelSerializer):
+class AddressSerializer(serializers.ModelSerializer):
     class Meta:
         model = MutableAddress
 
 
-class OrderSerializer(ModelSerializer):
+class PaymentSerializer(serializers.Serializer):
+    payment_identifier = serializers.CharField(max_length=50)
+    amount_value = serializers.DecimalField(decimal_places=9, max_digits=36, allow_null=True)
+    currency = serializers.CharField(max_length=10, allow_null=True)
```

Comment:
Currency of a payment should not be serialized, since it can't be different than the currency of the order. To allow creating payments in different currency, make the foreign_amount_value and foreign_amount_currency serializable.

Btw. Do we really need to respecify every field of the model here? Doesn't seem very DRY. I'm especially concerned about the field parameters which could get updated on the model, but are very easily forgotten here.